app/events/[eventId]/participants/edit/page.tsx

```diff
@@ -10,36 +10,25 @@ export default function ParticipantUpdateForm({
 }: {
   params: { eventId: string };
 }) {
-  const [email, setEmail] = useState('');
-  const [token, setToken] = useState('');
+  const [participantId, setParticipantId] = useState('');
   const router = useRouter();
 
   const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
 
-    router.push(
-      `/events/${params.eventId}/participants/${email}?token=${token}`,
-    );
+    router.push(`/events/${params.eventId}/participants/${participantId}`);
   };
 
   return (
     <>
       <h3>Update your data</h3>
 
       <form className='w-full space-y-4' onSubmit={handleSubmit}>
-        <FormInput
-          type='email'
-          label='E-Mail'
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}
-          required
-        />
-
         <FormInput
           type='text'
-          label='Token'
-          value={token}
-          onChange={(e) => setToken(e.target.value)}
+          label='Participant Id'
+          value={participantId}
+          onChange={(e) => setParticipantId(e.target.value)}
```

Comment:
Shouldn't this be calling a separate function? wrapping it in a function should enable running on the server and the client.